platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -151,6 +151,10 @@ typedef struct {
 static tm_global_t *tm_glb;
 
 /* Forward function declarations. */
+
+static void signal_request(void);
+static void signal_request_done(void);
```

Comment:
Forward declarations are not needed as functions are defined before they are called.